### PR TITLE
Add a make target to branch-cut a single repo

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Use like so:

    make branch-cut ORG=my-org REPO=my-repo

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

**Comment by chancez** [comment]
/lgtm
Works for me

File: Makefile [hunk 1]
```diff
@@ -52,6 +52,11 @@ prow-config:
 	docker pull registry.svc.ci.openshift.org/ci/determinize-prow-config:latest
 	docker run -it -v "${CURDIR}/core-services/prow/02_config:/config" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --prow-config-dir /config
 
+branch-cut:
+	docker pull registry.svc.ci.openshift.org/ci/config-brancher:latest
+	docker run -it -v "${CURDIR}/ci-operator:/ci-operator" registry.svc.ci.openshift.org/ci/config-brancher:latest --config-dir /ci-operator/config --org=$(ORG) --repo=$(REPO) --current-release=4.3 --future-release=4.4 --bump-release=4.4 --confirm
```
Comment (bparees) [inline Makefile]: should these release #s be parameters also?

Comment (stevekuznetsov) [reply Makefile]: No. Changing what the release is is a cut over task. I want to give people the option to cut over within the boundary of the current release.

**Pull request merged**